src/serverCommunication/index.ts

```diff
@@ -1,14 +1,15 @@
 import axios from 'axios';
 import Base from '../baseApi';
-import responseHandlers from '../utils/response/responseHandlers';
 import { ServerCommunication as Model } from './types';
+import { toApiResponse } from '../utils/response/responseHandlers';
+import { ApiResponse } from '../utils/response/apiResponse';
 
 const resourceName = 'serverCommunication';
 
-export default class ServerCommunication extends Base {
+export default class ServerCommunicationEndpoint extends Base {
   read(account: string) {
     this.setAccount(account);
-    const response = axios.get<Model>(`${this.getApiPath()}${resourceName}/read`, { withCredentials: false });
-    return responseHandlers.toApiResponse(response);
+    const response = axios.get<ApiResponse<Model>>(`${this.getApiPath()}${resourceName}/read`, { withCredentials: false });
```

Comment:
Granted, it was already like this, but: what's the reason for using axios directly in here, instead of `this._get()`?